lib/services/nt4_client.dart

```diff
@@ -326,7 +326,7 @@ class NT4Client {
     NT4Subscription newSub = NT4Subscription(
       topic: topic,
       uid: getNewSubUID(),
-      options: NT4SubscriptionOptions(periodicRateSeconds: period),
+      options: NT4SubscriptionOptions(periodicRateSeconds: period, all: true),
```

Comment:
Having the "all" subscription option be true by default can waste bandwidth when initially connecting or adding new widgets.  The changes in the NT restructure branch make custom subscription options much easier, so it can modified accordingly after this is merged into main. For now it would make more sense to have a separate method that subscribes all to a topic.